test/functions/generic/poi/issue-19198.good

```diff
@@ -0,0 +1,6 @@
+issue-19198.chpl:2: In function 'genericfunc':
+issue-19198.chpl:3: error: ambiguous call 'helper1()'
+issue-19198.chpl:18: note: candidates are: helper1()
+issue-19198.chpl:19: note:                 helper1()
+issue-19198.chpl:13: note:                 helper1()
```

Comment:
With the disclaimer that POI surprises me all the time and is in the name of the test's path here:  My intuition would be that only the overloads and 18 and 19 would win over 13 here, s.t. I was surprised that 13 showed up in the output.  Or are these candidates before any filtering and ranking has been done (in the "all visible functions" sense), in which case it makes more sense (and could be helpful if you were surprised by this.

Is it worth having a clone of this test that sees what happens when the mcallFunc() path is called?  (I think here, we're never finding out because the nCallFunc() path issues an error first?  Or would we get both sets of errors if there was one?

Reply:
> Or are these candidates before any filtering and ranking has been done (in the "all visible functions" sense), in which case it makes more sense (and could be helpful if you were surprised by this.

Yes, we print out the candidates, not the "most specific candidates remaining after down selecting some".

I've just added a version that runs mCallFunc. It has the behavior I would expect (and runs). We wouldn't get both sets of errors if there were errors in both in this situation.

Reply:
Great, thanks!